Lesson15/Lesson15/yasin/script.js

```diff
@@ -0,0 +1,120 @@
+const arrayExercise9 = ['An', 'array', 'of', 'strings'];
+
+console.log('Exercise9.1: ', arrayExercise9.join(" "));
+console.log('Exercise9.2: ', arrayExercise9.join(""));
+console.log('Exercise9.2: ', arrayExercise9.join(" Excercise "));
+console.log('------------------------');
+// 10. Use the `replace` method to replace "can help" with "help" in the string stringExercise10 and print the result to the console.
+const stringExercise10 =
+	'Software design patterns can help simplify complex systems.';
+
+console.log('Exercise10: ', stringExercise10.replace("can help", "help"));
+console.log('------------------------');
+// 11. Use the `startsWith` method to check if strings below starts with a price, and print
+// the result to the console.
+const string1Exercise11 = '$9.99 - Sale! 50% off all items in store';
+const string2Exercise11 =
+	'Bundle deal: Save $50 when you buy two products together';
+const priceSymbol = '$';
+
+console.log('Exercise 11.1: ',string1Exercise11.startsWith(priceSymbol));
+console.log('Exercise 11.2: ',string2Exercise11.startsWith(priceSymbol));
+console.log('------------------------');
+// 12. Use the `endsWith` method to check if a string ends with a "!" (exclamation mark), and print the result to the console.
+const string1Exercise12 = '$9.99 - Sale! 50% off all items in store';
+const string2Exercise12 =
+	'Bundle deal: Save $50 when you buy two products together!';
+
+console.log('Exercise 12.1: ', string1Exercise12.endsWith("!"));
+console.log('Exercise 12.2: ', string2Exercise12.endsWith("!"));
+console.log('------------------------');
+// 13. Use the `includes` method to check if a string includes a "%" character and print
+// the result to the console.
+const string1Exercise13 = '$9.99 - Sale! 50% off all items in store';
+const string2Exercise13 =
+	'Bundle deal: Save $50 when you buy two products together!';
+const percentSymbol = '%';
+
+console.log('Exercise13: ', string1Exercise13.includes("%"));
+console.log('Exercise13: ', string2Exercise13.includes("%"));
+console.log('------------------------');
+// 14. Use the `substring` method to extract the word "Databases" from string stringExercise14
+// based on two indices and print it to the console.
+const stringExercise14 =
+	'Databases are an essential component of many applications.';
+
+console.log('Exercise 14: ', stringExercise14.substring(0, 8));
+console.log('------------------------');
+// 15. Remove extra spaces from the string stringExercise15 and print the result to the console.
+const stringExercise15 = '    Hello world    ';
+
+console.log('Exercise15: ', stringExercise15.trim());
+console.log('------------------------');
+// 16. Extract the price and currency from a string below using "slice" method and print it to console.
+const planPriceString = 'Premium plan - 9.99 USD/month.'; // Should return "9.99 USD"
+
+console.log('Exercise16: ', planPriceString.slice(15, 23));
```

Comment:
Same as in ex. 14, we need to implement a dynamic solution here, so you should not use hardcoded indexes. 
Instead, rely on indexOf method to find the start and end position for the slice method.